Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -29,58 +29,76 @@ @implementation AppCenterDelegateObjC
 - (BOOL)isAppCenterEnabled {
   return [MSAppCenter isEnabled];
 }
+
 - (void)setAppCenterEnabled:(BOOL)isEnabled {
   return [MSAppCenter setEnabled:isEnabled];
 }
+
 - (NSString *)installId {
   return [[MSAppCenter installId] UUIDString];
 }
+
 - (NSString *)appSecret {
 #if GCC_PREPROCESSOR_MACRO_PUPPET
   return [[MSAppCenter sharedInstance] appSecret];
 #else
   return @"Internal";
 #endif
 }
+
 - (NSString *)logUrl {
 #if GCC_PREPROCESSOR_MACRO_PUPPET
   return [[MSAppCenter sharedInstance] logUrl];
 #else
   return @"Internal";
 #endif
 }
+
 - (NSString *)sdkVersion {
   return [MSAppCenter sdkVersion];
 }
+
 - (BOOL)isDebuggerAttached {
   return [MSAppCenter isDebuggerAttached];
 }
+
 - (void)setCustomProperties:(MSCustomProperties *)customProperties {
   [MSAppCenter setCustomProperties:customProperties];
 }
 
+- (void)startAnalyticsFromLibrary {
+  [MSAppCenter startFromLibraryWithServices:@[ [MSAnalytics class] ]];
```

Comment:
Not a big deal but can you take a list of services as a parameter?
For now we only have Analytics that can be started from library but let's make is flexible.

Reply:
The problem is we cannot directly refer SDK services from shared code when we are building it as puppet (without introducing special bridge header). Even more, I believe that we shouldn't do it because then there would be no point in splitting into multiple applications.

Reply:
I agree with Ivan.

Reply:
I see.